progress/angular/CHANGELOG.md

```diff
@@ -1,13 +1,20 @@
-# Recent Changes
+# Changelog
+
+## v1.1.5
```

Comment:
This will be 1.2.0 (bug fixes, minor improvements = patch, new feature = minor, breaking changes or major feature = major)